tests/qadence/test_measurements/test_sample_exp.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import annotations
+
+import pytest
+import torch
+from torch import Tensor
+
+from qadence import (
+    AbstractBlock,
+    QuantumCircuit,
+    QuantumModel,
+    add,
+    chain,
+    kron,
+)
+from qadence.measurements import Measurements
+from qadence.measurements.samples import compute_expectation
+from qadence.operations import CNOT, RX, Z
+from qadence.types import BackendName
+
+
+@pytest.mark.parametrize(
```

Comment:
I would try to iterate over backends to check if there isn't any issue. Also, I would try to use the random circuit/observable generation functionality in `strategies.py`. I think it would make for a better quality test.

Reply:
Opening an issue for this [Add test cases for twirl error mitigation to test multiple backends and strategies](https://github.com/pasqal-io/qadence/issues/348)